website/docs/docs/use-dbt-semantic-layer/sl-cache.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Cache common queries"
+id: "sl-cache"
+description: "Cache common queries to speed up performance and reduce query computation."
+tags: [Semantic Layer]
+sidebar_label: "Cache common queries"
+---
+
+
+The dbt Semantic Layer allows you to cache common queries in order to speed up performance and reduce compute on expensive queries. 
+
+There are two layers of caching:
+
+- [**Result caching**](#result-caching) leverages your data platform's built-in caching layer.
+- [**Declarative caching**](#declarative-caching) allows you to pre-warm the cache using saved queries.
+
+## Result caching
+
+Result caching leverages your data platform’s built-in caching layer and features. [MetricFlow](/docs/build/about-metricflow) generates the same SQL for multiple query requests, this means it can take advantage of your data platform’s cache.
+
```

Comment:
is this right Jordan?

Reply:
for last sentence, not sure if that's across the board for all platforms. ask users to double check their data platform specs